src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -664,9 +665,22 @@ export default class AircraftCommander {
     runIls(aircraft, data) {
         const approachType = 'ils';
         const runwayName = data[1].toUpperCase();
-        const runway = AirportController.airport_get().getRunway(runwayName);
+        const runwayModel = AirportController.airport_get().getRunway(runwayName);
+        const glideslopeElevation = runwayModel.getGlideslopeAltitudeAtFinalApproachFix();
+        const minmiumGlideslopeInterceptAltitude = _ceil(glideslopeElevation, -2);
 
-        return aircraft.pilot.conductInstrumentApproach(approachType, runway);
+        if (aircraft.mcp.altitude < glideslopeElevation) {
```

Comment:
You could move all the logic to the runwY and call a method here like `.isBelowMinimunGlideslopAltitude(currentAltitude)`

Reply:
essentially did now:
```
const minimumGlideslopeInterceptAltitude = runwayModel.getMinimumGlideslopeInterceptAltitude();

        if (aircraft.mcp.altitude < minimumGlideslopeInterceptAltitude) {
```